hm_01_03.js

```diff
@@ -0,0 +1,84 @@
+//Get days in year
+const countDays = (year) => {
+  if ((year % 100 !== 0 && year % 4 === 0) || year % 400 === 0) {
+    return 366;
+  } else return 365;
+};
+countDays(2020);
+
+//Get day of a year
+const getDayNumber = (date) => {
+  const current = new Date(date);
+  const year = current.getFullYear();
+  const newYear = new Date(year, 0, 1);
+  let diff = Math.floor((current - newYear) / 86400000);
```

Comment:
86400000 numbers like this are better to store in const with self-explained name
for example 
`const msInDay = 86400000`